instana/agent-google-market-app
# Update with the latest helm chart and docker images

The helm chart and instana/agent and instana/leader-elector images used in the Instana Agent GCP app is sorely out of date.

We need to do the following:
- Update with the latest helm chart resources where applicable
- Ensure that the appropriate configuration values are also exposed in `schema.yaml` for the GCP Marketplace UI to consume
- Ensure that the following Docker images `instana/agent:1.0.28, `instana/agent:latest` and `instana/leader-elector:0.5.4` images are uploaded and available in GCR
- Test this end to end
